<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyAttendance</class>
 <widget class="QWidget" name="DailyAttendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>52</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="titleBarGroup">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_min">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>.萍方-简</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
   color:white;
}
QPushButton:hover{
  background-color:rgb(85, 170, 255);
  color:rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>.萍方-简</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
   color:white;
}
QPushButton:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房情书体（非商用）</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开启属于你的</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/gallery.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_day">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房行舟体（非商用）</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_week">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>周一</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_attendance">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>立即打卡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_attendAmount">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_title</zorder>
  <zorder>label_day</zorder>
  <zorder>label_month</zorder>
  <zorder>label_week</zorder>
  <zorder>btn_attendance</zorder>
  <zorder>label_attendAmount</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
